march_gait_selection/CMakeLists.txt

```
@@ -2,31 +2,30 @@ cmake_minimum_required(VERSION 2.8.3)
 project(march_gait_selection)
 
 find_package(catkin REQUIRED COMPONENTS
+    march_description
     march_shared_classes
     march_shared_resources
     std_msgs
 )
 
 catkin_python_setup()
 
-catkin_package(
-    CATKIN_DEPENDS
+catkin_package(CATKIN_DEPENDS
+    march_description
     march_shared_classes
     march_shared_resources
     std_msgs
 )
 
-catkin_install_python(PROGRAMS scripts/${PROJECT_NAME}_node
+catkin_install_python(PROGRAMS scripts/${PROJECT_NAME}_node test/run_tests
```

It is not necessary to install tests, at least it isn't in other packages
```suggestion
catkin_install_python(PROGRAMS scripts/${PROJECT_NAME}_node
```